Emmanuelaaron/math-magicians
# Logic Link

## Links the Logic

I have linked the logic and UI so that interactiveness is added to the calculator

### Tools used
- React
- Eslint
- Stylint 

### To test this branch Locally
- Clone this repo ```git clone git@github.com:Emmanuelaaron/math-magicians.git```
- Checkout to ths branch ```git checkout feature-3"
- Install all the dependencies ```npm install``
- Run the server ```npm start```
- The browser should be open and the UI of a calculator displayed
-  Calculator should be able to perform different operations
